packages/blockprotocol/cli/commands/help.js

```diff
@@ -0,0 +1,31 @@
+import commandLineUsage from "command-line-usage";
+
+// ********************* MANUAL ********************* //
+
+const usageGuide = [
+  {
+    header: "blockprotocol CLI",
+    content:
+      "CLI for the {magenta Block Protocol} API. See https://blockprotocol.org/docs",
+  },
+  {
+    header: "Commands",
+    content: [
+      { name: "{bold publish}", summary: "publishes a block to the Block Hub" },
+      { name: "{bold help}", summary: "prints this guide" },
+    ],
+  },
+];
+
+// ********************* SCRIPT ********************* //
+
+/**
+ * Print the main help guide
+ */
+const script = () => {
```

Comment:
Unless `script` is some established term you have borrowed upstream, I would try to avoid it. Existing scripts follow [specific rules](https://github.com/blockprotocol/blockprotocol/pull/189), which cannot be used in commands – these are different. Not sure what exact term to use, but something like `run` would not be too much off.

Reply:
https://github.com/blockprotocol/blockprotocol/pull/575/commits/bd8dd3436a6d588d763e6211c473151178f9d134